src/Microsoft.Framework.Logging.Testing/project.json

```diff
@@ -1,10 +1,10 @@
 {
     "version": "1.0.0-*",
     "dependencies": {
-        "Microsoft.Framework.Logging.Interfaces": "1.0.0-*"
+      "xunit.runner.aspnet": "2.0.0-aspnet-*",
+      "Microsoft.Framework.Logging.Interfaces": "1.0.0-*"
     },
     "frameworks": {
-        "net45": { },
```

Comment:
@davidfowl I add this back along with new dependencies in new iteration, please take a look...I initially removed it after discussing with Brennan as this is the only project in Testing repo which has `net45` and wondered if it was intentional